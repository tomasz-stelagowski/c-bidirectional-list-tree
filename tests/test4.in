ADD_NODE 0
ADD_NODE 1
ADD_NODE 2
ADD_NODE 3
ADD_NODE 4
ADD_NODE 5
ADD_NODE 3
ADD_NODE 2
ADD_NODE 7
ADD_NODE 9
DELETE_NODE 7
RIGHTMOST_CHILD 3
DELETE_NODE 3
RIGHTMOST_CHILD 2
DELETE_NODE 8
RIGHTMOST_CHILD 2
DELETE_SUBTREE 9
RIGHTMOST_CHILD 2
ADD_NODE 4
ADD_NODE 4
ADD_NODE 4
SPLIT_NODE 4 11
RIGHTMOST_CHILD 4
DELETE_NODE 14
RIGHTMOST_CHILD 4
DELETE_NODE 4
DELETE_NODE 2
DELETE_NODE 13
RIGHTMOST_CHILD 1
SPLIT_NODE 1 5
RIGHTMOST_CHILD 15
RIGHTMOST_CHILD 1
SPLIT_NODE 5 6
RIGHTMOST_CHILD 5
RIGHTMOST_CHILD 6
RIGHTMOST_CHILD 16
DELETE_SUBTREE 15
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 0
