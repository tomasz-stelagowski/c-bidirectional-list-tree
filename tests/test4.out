OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
9
OK
8
OK
9
OK
4
OK
OK
OK
OK
14
OK
13
OK
OK
OK
12
OK
12
15
OK
16
-1
-1
OK
5
1
